modules/biobank/php/containercontroller.class.inc

```diff
@@ -0,0 +1,403 @@
+<?php declare(strict_types=1);
+/**
+ * This class features the code that enables access to the containers of the
+ * LORIS Biobank Module.
+ *
+ * PHP Version 7.2
+ *
+ * @category   Controller
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.text GPLv3
+ * @link       http://www.github.com/aces/Loris/
+ */
+namespace LORIS\biobank;
+
+/**
+ * This class features the code that enables access to the containers of the
+ * LORIS Biobank Module.
+ *
+ * PHP Version 7.2
+ *
+ * @category   Controller
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.text GPLv3
+ * @link       http://www.github.com/aces/Loris/
+ */
+class ContainerController
+{
+    /**
+     * Global variable $db
+     *
+     * @access private
```

Comment:
The `@access` comment doesn't serve much purpose given the `private` keyword right below it. (The comment also doesn't tell you much.. same with the variables below.)

Reply:
okay.